taxi/views.py

```diff
@@ -64,26 +67,56 @@
 
 class CarCreateView(LoginRequiredMixin, generic.CreateView):
     model = Car
-    fields = "__all__"
+    form_class = CarForm
     success_url = reverse_lazy("taxi:car-list")
 
 
 class CarUpdateView(LoginRequiredMixin, generic.UpdateView):
     model = Car
-    fields = "__all__"
+    form_class = CarForm
     success_url = reverse_lazy("taxi:car-list")
 
 
+class CarAssignView(LoginRequiredMixin, generic.UpdateView):
+    def post(self, request, *args, **kwargs):
+        car = Car.objects.get(pk=kwargs["pk"])
+        car.drivers.add(request.user)
+        return redirect("taxi:car-detail", pk=car.pk)
+
+
+class CarResignView(LoginRequiredMixin, generic.UpdateView):
+    def post(self, request, *args, **kwargs):
+        car = Car.objects.get(pk=kwargs["pk"])
+        car.drivers.remove(request.user)
+        return redirect("taxi:car-detail", pk=car.pk)
+
+
 class CarDeleteView(LoginRequiredMixin, generic.DeleteView):
     model = Car
     success_url = reverse_lazy("taxi:car-list")
 
 
 class DriverListView(LoginRequiredMixin, generic.ListView):
-    model = Driver
+    model = get_user_model()
```

Comment:
Code Style: According to the checklist, you should use `-` to split words in URL identification parameter `name`, not `_`. Please replace `driver_list` with `driver-list`.